.github/scripts/clean-build.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+source `dirname ${BASH_SOURCE[0]}`/config.sh
+
+if [[ "$DELETE_BUILD" = 1 ]]; then
```

Comment:
I'd move the check for `"$DELETE_BUILD" = 1` to the callers of this script and split it to individual scripts for individual components being built. Thay way those scripts can be used locally individually. The current way is quite dangerous as it deletes everything.